### ISSUE TITLE
Nothing works, nothing at all.

### ISSUE THREAD

**Issue by sabengry** [ankit/stylebot#649]
- The saved styles don't apply. 
- The saved styles appear to be empty.
- The backup has my data but downloading it and then importing it just erased the empty styles saved for those pages. 
- I cannot import previously saved backups from my computer. 
- Manually converting the JSON backups to CSS text and then pasting it into the style input does nothing. Does not replace the saved styles, does not even replace the now missing empty styles. 

In other words, nothing works, nothing at all.

**Comment by ron159** [comment]
same here
Edge 90.0.818.66

**Comment by sabengry** [comment]
I don't use Edge. The problem I have is in Chrome. After I managed to get the thing to work again, it has gone and deleted all my styles, again. 

And I am not getting notifications for this issue, as long as we are talking about nothing working.

**Comment by dallinmoak** [comment]
Seeing this issue on edge Version 92.0.902.67, but I wasn't having it earlier.

**Comment by Regan-Koopmans** [comment]
Also experiencing this issue in the latest version of Chrome.
